### ISSUE TITLE
Current command broken on cygwin

### ISSUE THREAD

**Issue by alexnv** [sdkman/sdkman-cli#378]
Doesn't work on cygwin enviroment on windows 7. I'm using babun.github.io/ 
Installation is ok, but command sdk current show following error:
```
{ ~ }  » sdk                                                                                    ~

Usage: sdk <command> [candidate] [version]
       sdk offline <enable|disable>

   commands:
       install   or i    <candidate> [version]
       uninstall or rm   <candidate> <version>
       list      or ls   [candidate]
       use       or u    <candidate> [version]
       default   or d    <candidate> [version]
       current   or c    [candidate]
       outdated  or o    [candidate]
       version   or v
       broadcast or b
       help      or h
       offline           <enable|disable>
       selfupdate        [force]
       flush             <candidates|broadcast|archives|temp>

   candidate  :  the SDK to install: groovy, scala, grails, akka, etc.
                 use list command for comprehensive list of candidates
                 eg: $ sdk list

   version    :  where optional, defaults to latest stable if not provided
                 eg: $ sdk install groovy

{ ~ }  » sdk c                                                                                ~ 1
__sdkman_path_contains:local:2: not valid in this context: (x86)/Intel/OpenCL
Files=''
Files=''
Files=''
Apps=''
Files=''
Files=''
Files=''
{ ~ }  »                                                                                      ~ 1

```

**Comment by alexnv** [comment]
The error in sdkman-utils.sh
need double quotes
```
 local exists="$(echo "$PATH" | grep "$candidate")"
```

**Comment by marc0der** [comment]
Okay, is that due to the presence of spaces in the `PATH` variable?

**Comment by alexnv** [comment]
Yes, that's right

**Comment by marc0der** [comment]
Want to raise a PR for that?

**Comment by alexnv** [comment]
What do you mean? (My English is very bad)

**Comment by marc0der** [comment]
A pull request. The change has been applied and has been released in 3.2.4.
